test/extensions/filters/http/common/compressor/compressor_integration_tests.h

```diff
@@ -8,6 +8,10 @@
 
 namespace Envoy {
 
+/*
+ Integration test to make sure that new logic, which does not care about content-length header
```

Comment:
This comment reads like a PR comment, not a code comment. Can we phrase this in a way that will age better?

For example, `Test verifying that we don't add any additional headers when the upstream doesn't respond with a content-length header` or something to that effect